src/lib/code-sample/python.ts

```diff
@@ -0,0 +1,22 @@
+import { snakeCase } from 'change-case'
+
+import type { CodeSampleDefinition } from './schema.js'
+
+export const createPythonRequest = (
+  request: CodeSampleDefinition['request'],
+): string => {
+  const parts = request.path.split('/')
+  const params = Object.entries(request.parameters)
+    .map(([key, value]) => `${snakeCase(key)}=${JSON.stringify(value)}`)
+    .join(', ')
+
+  return `seam${parts.map((p) => snakeCase(p)).join('.')}(${params})`
+}
+
+export const createPythonResponse = (
+  response: CodeSampleDefinition['response'],
+): string => {
+  const { body } = response
+  if (body == null) return 'None'
+  return JSON.stringify(body)
```

Comment:
We want to return what python returns (same with the function input). Hopefully this conversion step in each language is tractable. It's simple for JS because of the JSON functions.

We don't care about formatting here, only syntax, e.g., this would still be ok as we will plan to run there though a format step later.

```
ActionAttempt(    status='pending',
              action_type='UNLOCK_DOOR',
          action_attempt_id='11111111-2222-3333-4444-555555555555', result=None,
              error={}
)

Reply:
583378a1003ec0297539c8c0f694b6143cccc95e